packages/frontend-shared/cypress/support/e2e.ts

```diff
@@ -369,7 +369,9 @@ function visitLaunchpad () {
   return logInternal(`visitLaunchpad ${Cypress.env('e2e_launchpadPort')}`, () => {
     return cy.visit(`/__launchpad/index.html`, { log: false }).then((val) => {
       return cy.get('[data-e2e]', { timeout: 10000, log: false }).then(() => {
-        return val
+        return cy.get('.spinner', { timeout: 10000, log: false }).should('not.exist').then(() => {
```

Comment:
do we think these timeout increases are performance related to the app? Wonder if its worth creating an issue but I also think this might be pretty difficult to track down the why.

Reply:
I have no idea what's causing this to hang and whether it's an App issue or just an issue in our own tests and how we're setting things up. I've never seen this behavior in the App outside of these tests or gotten a report of an issue of this page hanging.